### PR TITLE
:bug: Improve chart-sync

### PR DESCRIPTION
**Pull request by Marigold**
Couple of bugfixes for `chart-sync`

- Don't use default grapher schema `004`, but always load the schema specified in `config["$schema"]` (otherwise we get errors in chart-diff)
- Remap variableId of `sortColumnSlug`, this was previously left out (also try mapping new unknown fields)

### CODE REVIEW

File: apps/wizard/app_pages/chart_diff/conflict_resolver.py [hunk 1]
```diff
@@ -153,7 +152,7 @@ def resolve_conflicts(self, rerun: bool = False):
                 config.pop(field, None)
 
             # Verify config
-            config_new = validate_chart_config_and_set_defaults(config, schema=get_schema("004"))
+            config_new = validate_chart_config_and_set_defaults(config, schema=get_schema_from_url(config["$schema"]))
```
Comment (lucasrodes) [inline apps/wizard/app_pages/chart_diff/conflict_resolver.py:155]: could it be the case that this raised a `KeyError` exception bc `$schema` is not a property in `config`?

**Review by lucasrodes — APPROVED** [review]
lgtm, thanks!

File: apps/wizard/app_pages/indicator_upgrade/charts_update.py [hunk 2]
```diff
@@ -105,7 +105,7 @@ def push_new_charts(charts: List[gm.Chart], schema_chart_config: Dict[str, Any])
             config_new = update_chart_config(
                 chart.config,
                 st.session_state.indicator_mapping,
-                schema_chart_config,
+                get_schema_from_url(chart.config["$schema"]),
```
Comment (lucasrodes) [inline apps/wizard/app_pages/indicator_upgrade/charts_update.py:108]: also, possible `KeyError` raised sometimes?

File: apps/wizard/app_pages/chart_diff/conflict_resolver.py (hunk 1 above)
Comment (Marigold) [reply apps/wizard/app_pages/chart_diff/conflict_resolver.py:155]: Good question. Fortunately, `$schema` is never null.

**Pull request merged**